### ISSUE TITLE
Create a RMD document with raincloud plots and descriptive statistics for each variable in each dataset

### ISSUE THREAD

**Issue by GForb** [GForb/prediction_for_autistic_children#3]
Quatro (formally known as Rmarkdown - they are basically the same thing) allows you to create documents from R code. You can output a number of formats - for this html will be simplest

For each study create a document which summarised and plots all the variables. You can use your raincloud plot function for the plots and the code from the gui that creates a summary table for summary statistics (it might be worth turning this into a function)

To turn the dataframe of summary statsitics into a table you can use packages such as kable, flextable or [gt](https://github.com/rstudio/gt)

For now just include age, gender and parent report SDQs (total and subdomains).

Create a separate report for each study and a combined one for all studies (have study name as a level one header using "#")

**Comment by GForb** [comment]
We have switched to Rmarkdown as it allows easy specification of a different output directory to the directory that the .rmd file is saved. This is necessary as we'd like to save any outputs from the project in a separate place to any code.

This is not currently supported in quarto - see these threads for discussion:

https://github.com/quarto-dev/quarto-r/issues/89#issuecomment-1446862648
https://github.com/quarto-dev/quarto-r/issues/81

**Comment by GForb** [comment]
Current issue with this is that the tables do not appear properly in the document but instead one after the other.

**Comment by GForb** [comment]
to fix this I followed the advice in this post: https://stackoverflow.com/questions/66812874/printing-gt-table-inside-a-loop-with-rmarkdownrender-not-working

It involved adding htmltools::tagList() to the table and then printing. I then had to add results=`asis` to the chunk options for that particular chunk to let rmarkdown know that it is getting html straight.

I do not really know what is going on here but it works.
